pkgs/servers/syncserver/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, python2
+, fetchFromGitHub
+}:
+
+let
+  python = python2.override {
+    packageOverrides = self: super: {
+      # Older version, used by syncserver, tokenserver and serversyncstorage
+      cornice = super.cornice.overridePythonAttrs (oldAttrs: rec {
+        version = "0.17";
+        src = oldAttrs.src.override {
+          inherit version;
+          sha256 = "1vvymhf6ydc885ygqiqpa39xr9v302i1l6nzirjnczqy9llyqvpj";
+        };
+      });
+    };
+  };
+
+# buildPythonPackage is necessary for syncserver to work with paster scripts
+in python.pkgs.buildPythonPackage rec {
+  pname = "syncserver";
+  version = "1.8.0";
+  disabled = ! python.pkgs.isPy27;
+
+  src = fetchFromGitHub {
+    owner = "mozilla-services";
+    repo = "syncserver";
+    rev = version;
+    sha256 = "0hxjns9hz7a8r87iqr1yfvny4vwj1rlhwcf8bh7j6lsf92mkmgy8";
+  };
+
+  buildInputs = with python.pkgs; [ unittest2 ];
```

Comment:
```suggestion
  checkInputs = with python.pkgs; [ unittest2 ];
```

Reply:
It cannot build without `unittest2`, even if `doCkeck` is false, which is why it's in `buildInputs` and not `checkInputs`.

Reply:
I can't reproduce this.

Reply:
Me neither actually. I must have been mistaken.